gnosis/safe/api/transaction_service_api/transaction_service_api.py

```diff
@@ -305,6 +305,37 @@ def remove_delegate(
             raise SafeAPIException(f"Cannot remove delegate: {response.content}")
         return True
 
+    def remove_delegate_signed(
```

Comment:
I don't like to have two functions to remove delegates. We have here two options in my opinion. 
- Remove the function that is signing inside
- or remove_delegate with optional signature and optional signature parameters as the signer. 
I prefer the first one, because in my opinion calculate the signature is out of the goal of a client API function.
For both options we should set the label breaking_change for the issue.

Reply:
Ok, makes sense. Removed here https://github.com/safe-global/safe-eth-py/pull/981/commits/a7eb6923bf34637721a047a1946c4062057f4a15